.circleci/config.yml

```diff
@@ -282,7 +306,7 @@ jobs:
           name: Install Datadog OpenTracing nginx module
           command: |
             CODENAME=$(lsb_release -c | cut -f2) # eg "bionic", "xenial"
-            apt install -y ./packages/datadog-opentracing-nginx*${CODENAME}*.deb
+            apt install -y ./artifacts/datadog-opentracing-nginx*${CODENAME}*.deb
             ldconfig
```

Comment:
It'd be nice if ldconfig wasn't necessary after installation.
Can we do things to make it not necessary?

Reply:
Known issue, I've just not yet added lfconfig to package postinst (eg https://people.canonical.com/~cjwatson/ubuntu-policy/policy.html/ch-sharedlibs.html )